htdocs/core/lib/functions.lib.php

```diff
@@ -82,15 +82,15 @@ function utf8_decode($elements)
 function getMultidirOutput($object, $module = '')
 {
 	global $conf;
-	if (!is_object($object)) {
+	if (!is_object($object) && empty($module)) {
```

Comment:
In most cases, $module is empty and is guessed from $object. Idea of @frederic34 is to have only $object as parameter ($module is for special cases and will be always empty)